packages/cashier/src/components/page-container/page-container.scss

```diff
@@ -13,8 +13,41 @@
     &__content {
         min-height: 100%;
         display: flex;
+        flex: 1;
+    }
+
+    &__left {
```

Comment:
Can't we merge `__left` and `__right` classes? the only difference is padding, so I think it's better to just have a class for modify one of the padding and remove all the duplicates.